src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -276,7 +276,7 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
       # Very light checking of the config file.
       self.assertEqual(len(config['registrationRequests']),
                         len(config['grantRequests']))
-      # check that the codnfig file contains consistent PAL&PPA data 
+      # check that the config file contains consistent PAL&PPA data 
```

Comment:
Please see my other comment -- I don't think we can actually require that the CBSDs on the cluster list get a PAL grant: the CBSD could also ask for a GAA grant on a non-PAL channel and that would be valid.

Reply:
I replied to your previous comment, I will copy it here: 

the checks here are to verify the following:
- the PAL channels of one PPA form a frequencyRange correctly.
- if the CBSD is in the cluster list of a PPA then the grant frequency range should be completely inside the frequency range of the PPA or completely outside of it, so it should not partially overlap with it.

don't you agree with that?

Reply:
Sorry, I must have missed that comment somehow. I'll have to also admit now that I did not read the code super carefully because I assumed I knew what it was doing (sorry!).

I think the checks you've described are valid. I'd suggest using the `msg` param of the assert functions to tell the tester what's wrong with the config, which will serve the dual purpose of telling anyone who's reading the code what it's supposed to do (more quickly than reading the code).